Mathlib/Tactic/Continuity.lean

```diff
@@ -0,0 +1,29 @@
+/-
+Copyright (c) 2023 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll
+-/
+
+import Aesop
+
+/-!
+# Continuity
+
+We define the `continuity` tactic using `aesop`. -/
+
+declare_aesop_rule_sets [Continuous]
+
+/--
+The `continuity` attribute used to tag continuity statements for the `continuity` tactic. -/
+macro "continuity" : attr =>
+  `(attr|aesop safe apply (rule_sets [$(Lean.mkIdent `Continuous):ident]))
```

Comment:
I think this is fine for now. However, without backtracking there may be cases where we won't want `Continuous.comp` to trigger (e.g., in situations like maps into subtypes where you want to use `continuous_induced_rng` and you conclude continuity from the fact that the composition is continuous and reduces somehow; this wasn't marked with `continuity` before though). In any case, this should be fixable in the future with `macro_rules` that allow us to set various `aesop` options, or else those could be manually applied when necessary.